src/main/java/pl/tomaszdziurko/jvm_bloggers/JvmBloggersConfiguration.java

```diff
@@ -33,4 +34,8 @@ public Client getMailingRestClient(@Value("${mailing.apiKey}") String malingApiK
         return client;
     }
 
+    @Bean
+    public CacheManager cacheManager() {
+        return new GuavaCacheManager();
```

Comment:
@adamdec: Reinventing the wheel is not KISS IMHO. Why writing custom cache implementation based on `ConcurrentHashMap` if we can use Spring provided [Cache Abstraction](https://docs.spring.io/spring/docs/current/spring-framework-reference/html/cache.html) with Guava cache as implementation provider? We don't even have to add any additional dependencies, we already have it in the project. Where do you see overkill here? IMHO using Spring Cache Abstraction has many advantages over custom cache implementation.:
- writing custom cache (even simple map/get/put approach) involves writing custom code and every additional custom line of code increases probability of bugs and errors (opposition of KISS)
- _Spring Cache Abstraction_ can be used in a clean and declarative way, just add @Cacheable annotation and you got caching (in opposite to custom based implementation)